firebase-firestore/src/test/java/com/google/firebase/firestore/spec/SpecTestCase.java

```diff
@@ -14,6 +14,7 @@
 
 package com.google.firebase.firestore.spec;
 
+import static com.google.common.base.Strings.emptyToNull;
```

Comment:
We should try no to put in more Guava dependencies.

Reply:
In my opinion, and exception for tests makes a lot of sense since there are no downstream dependencies on the test artifacts that could result in dependency mismatch conflicts.  Moreover, this class already imports `com.google.common.collect.Sets` from guava, so this is not a net new dependency on guava.

Reply:
I am OK to import Guava in tests. 

And thanks for adding this, will be really handy to try to debug a specific test.